### PR TITLE
Add client scan benchmark.

### PR DESCRIPTION
**Pull request by petermattis**
Fix storage.Store to only start the underlying engine once when
initializing via Store.Bootstrap followed by Store.Start.

### CODE REVIEW

**Comment by bdarnell** [comment]
LGTM

File: server/testserver.go [hunk 1]
```diff
@@ -100,9 +102,16 @@ func (ts *TestServer) Start() error {
 		return util.Errorf("could not init server: %s", err)
 	}
 
-	ctx.Engines = []engine.Engine{engine.NewInMem(proto.Attributes{}, 100<<20)}
```
Comment (spencerkimball) [inline server/testserver.go]: This is going to conflict with my change.

File: storage/store.go [hunk 2]
```diff
@@ -367,9 +367,20 @@ func (s *Store) String() string {
 
 // Start the engine, set the GC and read the StoreIdent.
 func (s *Store) Start() error {
-	// Start engine (i.e. open and initialize RocksDB database).
-	if err := s.engine.Start(); err != nil {
-		return err
+	if s.Ident.NodeID == 0 {
```
Comment (spencerkimball) [inline storage/store.go]: Might want to add a comment as to what this would imply: "If NodeID != 0, then the store has already been started."

**Comment by spencerkimball** [comment]
LGTM

File: server/testserver.go (hunk 1 above)
Comment (petermattis) [inline server/testserver.go]: Yep. Fixed.

File: storage/store.go (hunk 2 above)
Comment (petermattis) [inline storage/store.go]: Sure. Done.

**Pull request merged**